admin-notice/admin-notice.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Automattic\VIP\Admin_Notice;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+require_once __DIR__ . '/class-admin-notice.php';
+require_once __DIR__ . '/class-admin-notice-controller.php';
+require_once __DIR__ . '/conditions/interface-condition.php';
+require_once __DIR__ . '/conditions/class-date-condition.php';
+
+$admin_notice_controller = new Admin_Notice_Controller();
+$admin_notice_controller->init();
```

Comment:
What advantage is it to call `init()` here, rather than moving the `add_action()` call here and removing all WordPress-defined function calls out of it, to just have a plain PHP class?

It's a controller - it shouldn't care what application it is part of as it is just moving data around.

Reply:
done